src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/reader/ClusterDetailsEventProcessor.java

```diff
@@ -75,15 +87,18 @@ public void processEvent(Event event) {
 
     // An example node_metrics data is something like this for a two node cluster:
     // {"current_time":1566414001749}
+    // 1566414001749
     // {"overrides": {"enabled": {}, "disabled": {}}
```

Comment:
This is not valid json: we are missing a `}` in the end

Reply:
Added.